cfme/cloud/stack.py

```diff
@@ -97,7 +98,8 @@ def edit_tags(self, tag, value):
     def get_tags(self):
         navigate_to(self, 'Details')
         row = sel.elements("//*[(self::th or self::td) and normalize-space(.)={}]/../.."
-                     "//td[img[contains(@src, 'smarttag')]]".format(quoteattr("My Company Tags")))
+                     "//td[img[contains(@src, 'smarttag')]]".format(
+            quoteattr("My Company Tags"))) if current_version() < '5.8' else "//td[i[contains(@class, 'fa-tag')]]"
```

Comment:
honestly, I'd ask you to use self.appliance.current_version but this file anyway has to be refactored